enclave-manager/web/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
```

Comment:
@Dartoxian do we need to remove this file?

Reply:
ah thanks for the flag, yep I'll tidy this up.